cli/cmd/tapRunner.go

```diff
@@ -57,6 +58,16 @@ func RunMizuTap() {
 		}
 	}
 
+	var contract string
+	if config.Config.Tap.ContractFile != "" {
+		bytes, err := ioutil.ReadFile(config.Config.Tap.ContractFile)
+		if err != nil {
+			logger.Log.Errorf(uiUtils.Error, fmt.Sprintf("Error reading contract file: %v", errormessage.FormatError(err)))
+			return
+		}
+		contract = string(bytes)
```

Comment:
Can you test here that contract is a valid OAS and if it is not exit with an error message?

Reply:
@nimrod-up9 done with https://github.com/up9inc/mizu/commit/eda90276e25e711abf37bfb6df5a077665c65d98